Framework/Crystal/src/AnvredCorrection.cpp

```diff
@@ -432,61 +451,61 @@ double AnvredCorrection::getEventWeight(double lamda, double two_theta) {
  *
  *       @param twoth scattering angle
  *       @param wl scattering wavelength
+ *       @param muRTooLarge bool to warn in muR limit exceeded
  *       @returns absorption
  */
-double AnvredCorrection::absor_sphere(double &twoth, double &wl) {
-  int i;
-  double mu, mur; // mu is the linear absorption coefficient,
-  // r is the radius of the spherical sample.
-  double theta, astar1, astar2, frac, astar;
-  //  double trans;
-  //  double tbar;
-
-  //  For each of the 19 theta values in dwiggins (theta = 0.0 to 90.0
-  //  in steps of 5.0 deg.), the astar values vs.mur were fit to a third
-  //  order polynomial in excel. these values are given in the static array
-  //  pc[][]
-
-  mu = m_smu + (m_amu / 1.8f) * wl;
-
-  mur = mu * m_radius;
+double AnvredCorrection::absor_sphere(const double twoth, const double wl, bool &muRTooLarge) {
+  //  For each of the 19 theta values in (theta = 0:5:90 deg)
+  //  fitted ln(1/A*) = sum_{i=1}^{N} pc[i][ith]*(muR)^i
```

Comment:
hmm, because A*(muR=0) = 1 I didn't fit that coeficient, but I had to include the row of 0s in the pc array so I could evaluate the polynomial in the loop, so probably is clearer to have it as i=0 -> N!

Reply:
OK I see now. You can leave this as it is - or add a comment that pc[0][ith]=0 so doesn't need fitting